### DISCUSSION TITLE
Host X/Twitter Spaces ......

### DISCUSSION THREAD

**Discussion by jimmy6DOF** [omigroup/media#92] [Ideas]
It looks like X/Twitter made feature parity a think between desktop and mobile.  Previously hosting a space was only possible through mobile apps and due to some auth and the shared nature of the account this was not part of our thinking although discussed on several occasions.  Now it can be.  

Done right an Open Metaverse backed open space could extend the reach to a new community of potential contributors to the active workgroups since there is a fair overlap between Metaverse in general and the X/Twitter crowd so we should be doing this collaboratively as a group in line with the mission to amplify member voices and support a wider open metaverse in general. 

THINKING FROM DISCORD:
folks I am kind of linking this work in motion we have going in X/Twitter Spaces and I want to double down on it as a theme  for a while see how it goes.   My reasoning is it fits with a lot of our goals for community while reducing a lot of the friction we have had producing regular standalone in-world events.  Quick pro & con table :

Pro:
Meet people where they are:  increased walk-in impulse audience who just happen to be on the platform and see the space is live and join (...... and find it is of real interest and thereby discover OMI!?)
Low Friction (if you have a x-twitter account)
Easier to get speakers for same reason(s)
Can arrange an afterhours 3D world and promote it in the session with links

Con:
Single Platform Lockin and all that baggage
Audio only

 .....applying a quick Ben Franklin ruler to the list seems like it's a winner but happy to hear alternative perspectives before booking any speakers

**Comment by indiebio** [comment]
I fully support this, and great that Spaces is now more accessible to Desktop.

**Comment by jimmy6DOF** [comment]
Following the success of Khronos/glTF and RP1 spaces and the group's agreement to lean into this a format for wider discussion here is a proposal for a speaker and a clod call intro for consideration.  Kind of an Aim High goal but we could get lucky and the template might be useful to craft other invites in the future. 

INVITEE:  [Matthew Ball](https://x.com/ballmatthew)
Probably needs no introduction around here given he wrote the authoritative book on the Metaverse and set up the first Metaverse related Index Fund on Wall St.  His canonical 2021 book (link) was recently re-released with extensive updates and Mr.Ball has been doing the rounds (link)(link(link).   It is a long shot but may be possible that he is interested in the low friction mass communication live roundtable open forum type experience available through Spaces as we propose so it can't hurt to ask ...... 
  

INTRO (to be send from OMI X/Twitter account by Private Message)  :

> OMI is a public interest group dedicated to the Metaverse since April 2021 and we would like to invite you to a public discussion (x.com-Space) about your recent updated to the main book which was such a milestone in 2021.    
> 
> Needless to say we are all super fans and share an enthusiasm for the Metaverse as-is and as it is evolving.     
> 
> We have approximately 200 active members from a loose community of 5000 through our Discord, GitHub, X/Twitter, and W3C News Letter (we are an affiliated interest group under W3C).     
> 
> OMI believes in openness and building in public and find the casual nature of  X.com/Twitter "spaces" event programs allows the best participation level where multiple speakers can actively join the conversation (with a level of moderation oversight we provide).  
> 
> If your interested at all or even just exploring the topic further to see if you are -- then we're ready to send more details on how we usually do it/promote it, agree on a loose topic list, and sort the logistics as needed.  We are completely flexible to your schedule and any other considerations so even though this is just a shot in the dark maybe the timing is right to promote the book again and, as birds of a feather, maybe OMI is a good fit for helping you do that !!  We would sure love to. 

------------------


Comments and constructive criticism is appreciated
... lazy links to be filled in later

**Reply by indiebio** [reply]
I'd enjoy a more critical view, rather than a fawning listen to a guru vibe. Kim had many comments on his definition, on a Twitter thread which I have lost, again. He also has this blogpost: https://kim.nevelsteen.com/2021/10/05/metaverse-interoperability-space/
It will be interesting to hear how his views have changed, if it's more aligned with Kim's now. I'm not sure Kim would want to be in a chat with him (he's not a fan), but we can ask. I think my concern is the tension between having a famous person who has a lot of reach but maybe more of a superficial understanding of the metaverse (as an investor, not a deep tech guy, if I understand it correctly), vs caring for our community and their deeper understanding. First is probably seeing if Ball is interested. But something to ask the community is, who are better alternatives to talk about this, maybe get in the Space with him?

**Comment by jimmy6DOF** [comment]
INVITEE: [Wagner James Au](https://x.com/slhamlet)
Author, "Making a Metaverse That Matters" (Wiley). Covering & creating metaverse culture since 2003. 
also the [New World Notes](https://nwn.blogs.com/nwn/)

INTRO :
> Hello -- just wanted to reach out to you because OMI are starting to host a X/Twitter Space as part of our outreach program to extend the community of anyone and everyone interested in Metaverse related topics.
> 
> And we want to invite you, as a widely recognized expert involved from the very beginning of the modern Metaverse, to a freeform discussion about your book and blog posts with our members.   
> 
> X/Twitter Spaces are not exactly a 3D Metaverse but it allows casual low friction access to wide audience where multiple speakers can drop in and join the conversation in a loosely, but actively, moderated townhall style group Q&A forum.  It also meets our objective of openness and building in public.  
> 
> If you are interested then we can easily work to fit whatever works best for you in terms of the logistics of timing etc and any points you want us to cover in advance.  We will make announcements to the OMI community through our socials and generally try to promote the session as much as we can. 
> 
> Anyway this is an expression of our interest and if you would like to get into more detail please just hit reply we can take it from there !!
>
